assets/src/edit-story/components/helpCenter/useHelpCenter/index.js

```diff
@@ -175,15 +190,48 @@ export function useHelpCenter() {
     [store.actions]
   );
 
+  // Once app is hydrated, start persisting unreadTips
+  const { isHydrated, unreadTipsCount } = store.state;
+  useEffect(() => {
+    if (isHydrated) {
+      const local = localStore.getItemByKey(LOCAL_STORAGE_PREFIX.FTUE);
```

Comment:
You need to fix  all the references to `LOCAL_STORAGE_PREFIX.FTUE` to `LOCAL_STORAGE_PREFIX.HELP_CENTER`

Reply:
Ope, yeaup

Reply:
Thanks for catching